metadata/plants.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: "plants"
+type: "use"
+topic_type: field
+title: "Plants"
+summary: |
```

Comment:
@Deeptivarshney could you maybe add a sentence or two of description of the topic? (I know it's broad but..)